base_layer/core/src/base_node/comms_interface/local_interface.rs

```diff
@@ -33,39 +33,42 @@ use crate::{
     proof_of_work::{Difficulty, PowAlgorithm},
 };
 use futures::{stream::Fuse, StreamExt};
-use tari_broadcast_channel::Subscriber;
 use tari_service_framework::reply_channel::SenderService;
+use tokio::sync::broadcast;
 use tower_service::Service;
 
+pub type BlockEventSender = broadcast::Sender<BlockEvent>;
```

Comment:
tokio broadcast assumes that the item broadcast is cheap to clone. BlockEvent has a large memory footprint, so I think we should wrap it in an Arc.

Reply:
Good suggestion, I wrapped the BlockEvents in Arcs.